app/models/shortener/shortened_url.rb

```diff
@@ -78,7 +78,9 @@ def self.fetch_with_token(token: nil, additional_params: {})
   end
 
   def self.merge_params_to_url(url: nil, params: {})
-    params.try(:except!, *[:id, :action, :controller])
+    if params.respond_to?(:permit!)
+      params = params.permit!.to_h.with_indifferent_access.except!(:id, :action, :controller)
```

Comment:
Do you need the last `!` here?

Reply:
On `except!`? 
Maybe not… but it was there in the old version… and this code is taken directly from the latest version of the gem. Would prefer not to change anything away from their code if we don't have to